lightfm/_lightfm_fast.pyx.template

```diff
@@ -316,10 +317,20 @@ cdef inline void compute_representation(CSRMatrix features,
 
         representation[lightfm.no_components] += feature_weight * feature_biases[feature]
 
+cdef inline flt normalize_repr(flt *v_repr, int no_components) nogil:
```

Comment:
Can we call this `compute_norm`? I think it better reflects what this is actually doing.